packages/jest-runner/src/jest-plugins/mixin-jest-environment.ts

```diff
@@ -39,14 +39,25 @@ export function mixinJestEnvironment<T extends typeof JestEnvironment>(JestEnvir
         if (state.coverageAnalysis === 'perTest' && event.name === 'test_start') {
           const ns = (this.global[this.global.__strykerGlobalNamespace__] = this.global[this.global.__strykerGlobalNamespace__] ?? {});
           ns.currentTestId = fullName(event.test);
+          ns.hitCount = state.hitCount;
```

Comment:
I don't think we should reset the `hitCount` between tests. When Stryker's core calculates the `hitLimit`, it takes all tests into account.